src/archiefvernietigingscomponent/report/tests/test_utils.py

```diff
@@ -622,3 +626,246 @@ def test_looptijd_without_einddatum(self, m):
         loop_tijd = get_looptijd(zaak)
 
         self.assertEqual(4, loop_tijd)
+
+    @freeze_time("2021-05-05")
+    def test_report_creation_with_process_owner(self, m):
+        process_owner = UserFactory.create(
+            role__type=RoleTypeChoices.process_owner,
+            role__can_start_destruction=False,
+            role__can_review_destruction=True,
+            role__can_view_case_details=True,
+        )
+        destruction_list = DestructionListFactory.create(name="Winter cases")
+        DestructionListItemFactory.create(
+            destruction_list=destruction_list,
+            status=ListItemStatus.destroyed,
+            extra_zaak_data={
+                "identificatie": "ZAAK-1",
+                "omschrijving": "Een zaak",
+                "toelichting": "Bah",
+                "startdatum": "2020-01-01",
+                "einddatum": "2021-01-01",
+                "zaaktype": "https://oz.nl/catalogi/api/v1/zaaktypen/uuid-1",
+            },
+        )
+        DestructionListItemFactory.create(
+            destruction_list=destruction_list,
+            status=ListItemStatus.destroyed,
+            extra_zaak_data={
+                "identificatie": "ZAAK-2",
+                "omschrijving": "Een andere zaak",
+                "toelichting": "",
+                "startdatum": "2020-02-01",
+                "einddatum": "2021-03-01",
+                "zaaktype": "https://oz.nl/catalogi/api/v1/zaaktypen/uuid-2",
+            },
+        )
+        DestructionListReviewFactory.create(
+            destruction_list=destruction_list,
+            status=ReviewStatus.approved,
+            author=process_owner,
+            text="What a magnificent list!",
+        )
+
+        mock_service_oas_get(
+            m,
+            "https://selectielijst.oz.nl/api/v1",
+            "selectielijst",
+            oas_url="https://selectielijst.oz.nl/api/v1/schema/openapi.json",
+        )
+        mock_service_oas_get(
+            m,
+            "https://oz.nl/catalogi/api/v1",
+            "ztc",
+            oas_url="https://oz.nl/catalogi/api/v1/schema/openapi.json",
+        )
+        m.get(
+            url="https://oz.nl/catalogi/api/v1/zaaktypen/uuid-1",
+            json={
+                "selectielijstProcestype": "https://selectielijst.oz.nl/api/v1/procestypen/uuid-1"
+            },
+        )
+        m.get(
+            url="https://oz.nl/catalogi/api/v1/zaaktypen/uuid-2",
+            json={
+                "selectielijstProcestype": "https://selectielijst.oz.nl/api/v1/procestypen/uuid-2"
+            },
+        )
+        m.get(
+            url="https://selectielijst.oz.nl/api/v1/procestypen/uuid-1",
+            json={"nummer": 1},
+        )
+        m.get(
+            url="https://selectielijst.oz.nl/api/v1/procestypen/uuid-2",
+            json={"nummer": 2},
+        )
```

Comment:
I know that tests don't have to be DRY, but these mocks are set up for each test case, I think moving them into a separate method and reusing it will improve the readability of the tests